src/test/run-make/expand-derive/input_pp.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+// minimal junk
+#![feature(no_core)]
+#![no_core]
+
+struct A {
+    a: i32,
+}
+#[automatically_derived]
+#[allow(unused_qualifications)]
+impl ::clone::Clone for A {
+    #[inline]
+    fn clone(&self) -> A {
+        match *self {
+            A { a: ref __self_0_0 } => {
+                ::clone::assert_receiver_is_clone(&(*__self_0_0));
```

Comment:
Oh, so there is. I can migrate this test to `src/test/pretty` then. But
there are more run-make tests of the same class (pretty-expanded,
pretty-expanded-hygiene). I guess they can be migrated in a separate PR.

On Mon, Apr 25, 2016 at 2:04 PM, Eduard-Mihai Burtescu <
notifications@github.com> wrote:

> In src/test/run-make/expand-derive/input_pp.rs
> <https://github.com/rust-lang/rust/pull/31414#discussion_r60958975>:
>
> > +
> > +// minimal junk
> > +#![feature(no_core)]
> > +#![no_core]
> > +
> > +struct A {
> > +    a: i32,
> > +}
> > +#[automatically_derived]
> > +#[allow(unused_qualifications)]
> > +impl ::clone::Clone for A {
> > +    #[inline]
> > +    fn clone(&self) -> A {
> > +        match *self {
> > +            A { a: ref __self_0_0 } => {
> > +                ::clone::assert_receiver_is_clone(&(*__self_0_0));
>
> There's one .pp file in there which is the expected output for one test.
> If nobody has removed it yet for being painful to update, that is.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/rust-lang/rust/pull/31414/files/64d22d453d5b71b33b149dac3c28b2d626446d9f#r60958975>
>